pages/profile/ProfileForm.js

```diff
@@ -0,0 +1,135 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function ProfileForm({
+  initialProfile,
+  avatars,
+  onSave,
+  onCancel,
```

Comment:
this creates a new Page which is not perfect. Please move the function ProfileForm into the file index.js.